src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -328,22 +326,23 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
                     label={t("faultEventMenu.operationalFailureRate")}
                     rate={snsOperationalFailureRate}
                     selected={selectedRadioButton === RadioButtonType.Operational}
-                    classes={classes}
                   />
                 )}
                 <Box display={"flex"} flexDirection={"row"} alignItems="center">
                   <FormControlLabel
                     value={RadioButtonType.Manual}
-                    control={<Radio className={classes.black} />}
+                    control={<Radio className={classes.selected} />}
```

Comment:
see my comment above

Reply:
changed to: style={{ color: theme.main.black }}